apps/site/src/components/layout/Footer.astro

```diff
@@ -0,0 +1,16 @@
+---
+const year = new Date().getFullYear();
+---
+
+<footer class="border-t border-t-gray-200 mt-auto">
+  <div class="max-w-7xl mx-auto p-2">
+    <p>
+      &copy; <span id="footer-year">{year}</span> - Code under MIT. Content (images
+      and articles) under CC BY-NC 4.0.
+    </p>
```

Comment:
I wouldn't license every piece of content under `CC BY-NC 4.0`. I would keep it opt-in. I would rather link in the footer to our `LICENSE` file on GitHub.